test/controllers/task.list.controller.unit.ts

```diff
@@ -29,32 +33,44 @@ describe("Task list controller tests", () => {
   });
 
   it("Should show recordDate as next due date when filing after nextMadeUpToDate", async () => {
-    const expectedDate = toReadableFormat(validCompanyProfile.confirmationStatement!.nextMadeUpTo);
-    mockGetCompanyProfile.mockResolvedValueOnce(validCompanyProfile);
-    mocks.mockAuthenticationMiddleware.mockClear();
-    const url = TASK_LIST_PATH.replace(":companyNumber", COMPANY_NUMBER);
-    const response = await request(app).get(url);
-    expect(response.text).toContain(expectedDate);
+    if (validCompanyProfile.confirmationStatement === undefined) {
+      fail;
```

Comment:
you'll need to include the brackets after the fail function name to call it - fail();